fluentgen/src/main/java/com/azure/autorest/fluent/model/clientmodel/FluentLiveTestStep.java

```diff
@@ -0,0 +1,10 @@
+/*
+ * // Copyright (c) Microsoft Corporation. All rights reserved.
+ * // Licensed under the MIT License.
+ */
+
+package com.azure.autorest.fluent.model.clientmodel;
+
+public abstract class FluentLiveTestStep {
```

Comment:
interface or abstract class?

Reply:
Decided to use abstract class due to common field `description`.